cou/utils/juju_utils.py

```diff
@@ -175,7 +175,8 @@ class Application:
     origin: str
     series: str
     subordinate_to: list[str]
-    units: dict[str, Unit]
+    units: dict[str, Unit]  # subordainte apps does not have units, use subodinate_units instead
+    subordinate_units: list[SubordinateUnit]  # this is the units for subordinate apps
```

Comment:
**suggestion:** 

The SubordinateUnit is already exists in Unit.subordinates.

Is that possible we create a property `subordinate_units` on `Application` class instead of adding a new field?

Also SubordinateUnit in Unit can show the subordinate relationship between the units.

Reply:
Sorry, I am not sure what do you mean. Can you explain more?

`subordinate_units` is not the same as `Unit.subordinates`

Reply:
Yes, they are different. But you can generate the `subordinate_units` by iterator over units to get them.

Reply:
Maybe, but we need this information in the `Application` class and in `get_applications` method